### PR TITLE
Create sweet.py

### PR DESCRIPTION
**Pull request by wmcgee3**

### CODE REVIEW

File: sweet.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# stuff
```
Comment (wmcgee3) [inline sweet.py:1]: What is that supposed to do???

Comment (wmcgee3) [reply sweet.py:1]: nothin really, I just thought it was pretty :D

Comment (Patrickgp) [reply sweet.py:1]: but it looks good homie

**Review by Patrickgp — APPROVED** [review]
She aight.

**Pull request closed without merging**